source/PreferencesPanel.cpp

```diff
@@ -876,10 +876,13 @@ void PreferencesPanel::DrawPlugins()
 				top.Y() += sprite->Height() + 10.;
 			}
 
+			string text = plugin.CreateDescription();
+
 			WrappedText wrap(font);
 			wrap.SetWrapWidth(MAX_TEXT_WIDTH);
 			static const string EMPTY = "(No description given.)";
-			wrap.Wrap(plugin.aboutText.empty() ? EMPTY : plugin.aboutText);
+			text += plugin.aboutText.empty() ? EMPTY : plugin.aboutText;
+			wrap.Wrap(text);
```

Comment:
```suggestion
			wrap.Wrap(plugin.CreateDescription() + plugin.aboutText.empty() ? EMPTY : plugin.aboutText);
```
optional

but EMPTY should probably go into to the unnamed namespace

Reply:
The ternary operator has low precedence, so I'd do `wrap.Wrap(plugin.CreateDescription() + (plugin.aboutText.empty() ? EMPTY : plugin.aboutText));` just to be sure no shenanigans occur with that addition.